### ISSUE TITLE
Prices are not updated with the LethalQuantities mod

### ISSUE THREAD

**Issue by MorsusP** [Major-Scott/TerminalPlus#3]
Right away I want to write, the mod is of high quality and I like it a lot, but there is a problem that spoils the impression a bit.

The prices of moons are not updated if you use the **LethalQuantities** mod, which can change the prices of moons depending on the current moon. This is probably because the price is static and can't be updated?

**Comment by Major-Scott** [comment]
Hi! Have you set the prices in the TP config to -1? That’s supposed to
override my prices and grab whatever the unchanged value is. If you already
set them to -1 tho I’ll definitely take a look :)

**Comment by MorsusP** [comment]
yeah, it was already -1 there, and still the prices didn't change^_^

**Comment by MorsusP** [comment]
yeah, it was already -1 there, and still the prices didn't change^_^

**Comment by Major-Scott** [comment]
Sorry for the wait! I just updated the mod for v50 and this should hopefully be fixed now
